_posts/en/newsletters/2020-09-30-newsletter.md

```diff
@@ -0,0 +1,189 @@
+---
+title: 'Bitcoin Optech Newsletter #117'
+permalink: /en/newsletters/2020/09/30/
+name: 2020-09-30-newsletter
+slug: 2020-09-30-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a compiler bug that casts doubt on the
+safety of secure systems and explains a technique that can be used to more
+efficiently verify ECDSA signatures in Bitcoin. Also included are our regular
+sections with popular
+questions and answers from the Bitcoin StackExchange, announcements of
+releases and release candidates, and summaries of notable changes to
+popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Discussion about compiler bugs:** a [test][oconnor test] written
```

Comment:
@roconnor-blockstream has now written a short blog post about this too: http://r6.ca/blog/20200929T023701Z.html

Reply:
Thanks @sipa added a reference to the post.